build.gradle

```diff
@@ -1021,6 +1031,18 @@ project(':iceberg-spark3-runtime') {
     compile(project(':iceberg-nessie')) {
       exclude group: 'com.google.code.findbugs', module: 'jsr305'
     }
+
+    integrationImplementation 'org.apache.spark:spark-hive_2.12'
```

Comment:
Do we need a separate source set as test inherits other things?

Reply:
Yeah rather than excluding everything from tests (and the projects they depend on) I thought it would be much easier to instead only list the exact things we need explicitly. I think this also makes it clear that this is not a set of unit tests.

Reply:
(This also made it easier (at least for me) to setup the integration test task and have it depend on the output of the shadow Jar task.

Reply:
Makes sense, thanks for explaining!